README.md

```diff
@@ -27,7 +27,9 @@ sword.compile(
   "useClass": true,
   "useConstant": true,
   "useId": true,
-  "useQuery": true
+  "useQuery": true,
+  "minify": false
+
```

Comment:
Just asking, why is there an extra line here

Reply:
Just a typo I removed this line now